src/rgw/rgw_bucket.cc

```diff
@@ -1034,6 +1034,7 @@ int RGWBucket::check_bad_index_multipart(RGWBucketAdminOpState& op_state,
   RGWRados::Bucket::List list_op(&target);
 
   list_op.params.list_versions = true;
+  list_op.params.ns = ns;
```

Comment:
In recent master, 
#define RGW_OBJ_NS_MULTIPART "multipart"
list_op.params.ns = RGW_OBJ_NS_MULTIPART;

Macro is taken instead of local declaration.
This PR can be dropped.